tqp/newrelic.py

```diff
@@ -27,6 +27,9 @@ def _hook_send_newrelic_event(self, payload, success):
                 "topic": payload["topic"],
                 "queue_name": self.queue_name,
                 "success": str(success),
+                "attributes": payload["meta"].get("attributes", {})
```

Comment:
looks like `meta` is always set

Reply:
only when `with_meta` is set: https://github.com/4Catalyzer/tqp/pull/23/files#diff-ba5945493e7a28bddde9aa9f54dd499fR152